packages/edit-site/src/components/global-styles/preview.js

```diff
@@ -198,19 +199,10 @@ const StylesPreview = ( { label, isFocused, withHoverView } ) => {
 									overflow: 'hidden',
 								} }
 							>
-								<motion.div
-									style={ {
-										fontFamily: headingFontFamily,
-										fontSize: 65 * ratio,
-										color: headingColor,
-										fontWeight: headingFontWeight,
-									} }
-									animate={ { scale: 1, opacity: 1 } }
-									initial={ { scale: 0.1, opacity: 0 } }
-									transition={ { delay: 0.3, type: 'tween' } }
-								>
-									Aa
-								</motion.div>
+								<PreviewTypography
+									fontSize={ 65 * ratio }
```

Comment:
Not necessary for this PR, but an idea for future follow up could be to use `<PreviewTypography />` for the label below. It's using the same heading typography styles, so it could further clean up this file.

It might just need a tweak to the props, passing styles instead of `fontSize` and some default labels, e.g., 

```jsx
<PreviewTypography
	style={ {
		fontSize: 65 * ratio,
	} }
	variation={ variation }
/>
```